pt/core-libraries/email.rst

```diff
@@ -3,17 +3,6 @@ Email
 
 .. php:namespace:: Cake\Mailer
 
-.. note::
-    A documentação não é atualmente suportada pela lingua portuguesa nesta
-    página.
-
-    Por favor, sinta-se a vontade para nos enviar um pull request no
-    `Github <https://github.com/cakephp/docs>`_ ou use o botão
-    **Improve This Doc** para propor suas mudanças diretamente.
-
-    Você pode referenciar-se à versão inglesa no menu de seleção superior
-    para obter informações sobre o tópico desta página.
-
```

Comment:
I would prefer to keep the note till the file is fully translated. Some paragraphs are missing at the end of the document